akka-actor-tests/src/test/java/akka/util/LineNumberSpecCodeForJava.java

```diff
@@ -11,21 +11,20 @@
 
 public class LineNumberSpecCodeForJava {
 
-	// @FunctionalInterface // will be uncommented as soon as '-source 1.8' is set
 	public static interface F extends Serializable {
 		public String doit(String arg);
 	}
 
-	// public F f1() { // FIXME These are commented out until the build is switched to Java 8
-	// return (s) -> s;
-	// }
+	public F f1() {
+		return (s) -> s;
+	}
 
-	// public F f2() {
-	// return (s) -> {
-	// System.out.println(s);
-	// return s;
-	// };
-	// }
+	public F f2() {
+		return (s) -> {
+			System.out.println(s);
+			return s;
+		};
```

Comment:
that's a lot of spaces, I see that you just follow the existing style here, but please reformat with 2 spaces